src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -24,13 +24,16 @@
 
     private final String _BROKERS_IDS_PATH = "/brokers/ids";
 
-    @Autowired
     private ZooKeeperHolder zkFactory;
```

Comment:
Can be final now.
(Furthermore, you could remove that PostConstruct and do it all in the Constructor.)